src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Webui::Users::NotificationsController < Webui::WebuiController
   MAX_PER_PAGE = 300
-  VALID_NOTIFICATION_TYPES = ['read', 'reviews', 'comments', 'requests', 'unread'].freeze
+  VALID_NOTIFICATION_TYPES = ['read', 'reviews', 'comments', 'requests', 'unread', 'incoming_requests', 'outgoing_requests'].freeze
```

Comment:
not possible to transform it in symbols?

Reply:
In case it's possible I wouldn't do that in this PR, maybe in another one for refactoring.